topics/terraform/README.md

```diff
@@ -74,6 +74,17 @@
 -
 </b></details>
 
+<details>
+<summary>What is one reason why manual processes can be helpful?</summary><br><b>
+For learning a platform when first starting out
+</b></details>
+
+<details>
+<summary>Why is it advisable to avoid using manual processes when creating infrastructure at scale?</summary><br><b>
```

Comment:
```suggestion
<summary>Why is it advisable to avoid using manual processes when creating infrastructure at scale?</summary>
```
then as a separate line
```
<br><b>
```